compute/admin_guide/authentication/oidc.adoc

```diff
@@ -87,7 +87,6 @@ Enter the following callback URL:
 +
 *\https://<CONSOLE>:<PORT>/api/v1/authenticate/callback/oidc*, and then click *Save*.
 
-
 [.task]
 ==== Configure Prisma Cloud
```

Comment:
@rcaballeromx this is also configuration of Prisma Console, so it should be somehow connected to the configuration of the Console for Azure. (since this section also shows how to create the user / group in the console and then make sure you can connect with it to the Prisma Console)

Reply:
@grevach No, the section starting in line 91 is a subsection for the configuration of the Okta integration.

The Azure AD equivalent starts in line 177. 

Each integration has it's own section where customers create the users or groups needed for that integration to work.

To avoid confusion, I see two alternatives:

* Change the heading in line 91 to `==== Configure Okta as an Identity Provider` OR
* Change the heading in line 177 and the heading in line 91 to `==== Configure the Prisma Cloud Console`

The better alternative is the first because it provides better SEO. When the page is indexed by search engines, headings heavily influence the quality of search results and the heading "Configure the Prisma Cloud Console" won't be as effective. WDYT?

Reply:
@rcaballeromx ok, if this is the case then I agree using `==== Configure Okta as an Identity Provider` and `==== Configure Azure AD as an Identity Provider` is better.
However, I don't see an equivalent section for PingOne which is also one of the supported providers.

Reply:
@grevach I've made the change to the headings. As far as I could tell, the PingOne section only includes the configuration changes that take place on the PingOne side, and that's why it doesn't have an equivalent section. @Tomerit Is that observation correct?